packages/react/src/components/Modal/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Modal as ModalRoot } from './Modal';
+import { ModalContent } from './ModalContent';
+import { ModalFooter } from './ModalFooter';
+import { ModalHeader } from './ModalHeader';
```

Comment:
Suggest just removing the individual barrel files for each of the subcomponents and just importing them directly. 

Example:
`import { ModalContent } from './ModalContent/ModalContent';` etc…